core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -82,7 +82,7 @@ object ReassignPartitionsCommand extends Logging {
     }
   }
 
-  def generateAssignment(zkUtils: ZkUtils, opts: ReassignPartitionsCommandOptions) {
+  def generateAssignment(zkUtils: ZkUtils, opts: ReassignPartitionsCommandOptions): Map[TopicAndPartition, Seq[Int]] = {
```

Comment:
Now that we are returning the assignment, it's a bit weird to print the assignment to stdout. Perhaps we should let the caller do that.